site/src/css/custom.css

```diff
@@ -58,10 +66,108 @@
 }
 
 .navbar__link:hover:not(.navbar__link--active):not(.header-github-link) {
-  background-color: var(--uitk-color-gray-600);
+  background-color: var(--uitk-palette-navigate-background-hover-high);
   border-bottom: 2px solid var(--uitk-navigable-indicator-hover);
 }
 
```

Comment:
--ifm-toc-border-color should use --uitk-separable-tertiary-borderColor (the RHS border of the sidebar)